### DISCUSSION TITLE
method dataframe_to_grid is missing in new version

### DISCUSSION THREAD

**Discussion by Jimmy-KL** [Kitware/trame#161] [General]
I tried to learn the examples of trame. But I found in the version I'm using `trame==2.2.2` , method [trame.widgets.vuetify.dataframe_to_grid(dataframe, options={})](https://trame.readthedocs.io/en/latest/trame.widgets.vuetify.html#trame.widgets.vuetify.dataframe_to_grid) is missing because when I tried to run example [table.py](examples/00_howdoi/table.py), I got the following error:
![image](https://user-images.githubusercontent.com/114913030/205454003-3eeae66d-ac2c-467d-9711-53bbfb3d5df4.png)
This error also happened in example [00_dataframe-table.py](https://github.com/Kitware/trame/blob/master/examples/02_vuetify/00_dataframe-table.py).

**Comment by psavery** [comment]
I think this was fixed in [this commit for trame-vuetify](https://github.com/Kitware/trame-vuetify/commit/94675c6c0e5d9e8cc645219123e0616d81537f84).

Can you run `pip install -U trame-vuetify` and try again?

**Reply by Jimmy-KL** [reply]
After I upgraded trame-vuetify from version 2.0.1 to 2.0.2, the bug is fixed.
